test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -518,8 +518,15 @@ Feature: Bind an application to a service using annotations
                 name: binding-app-via-gvk-service
                 annotations:
                     service.binding/host_internal_db: path={.spec.host_internal_db}
+                    service.binding/environment: path={.spec.userLabels.environment}
+                    service.binding/dbCredentials: path={.status.data.dbCredentials}
             spec:
                 host_internal_db: internal.db.stable.example.com
+                userLabels:
```

Comment:
this scenario has a focus on something fully different, and hence I would not add broader its focus. Instead, let's add a separate test or find another which focus is on exposing binding data using annotations.

Reply:
Good point, thx @pedjak 
Updated a different scenario